lib/vast/CodeGen/DefaultStmtVisitor.cpp

```diff
@@ -945,7 +946,15 @@ namespace vast::cg
                     index_exprs.push_back(mk_value_builder(expr->getIndexExpr(index)));
                     break;
                 }
-                case clang::OffsetOfNode::Kind::Field:
+                case clang::OffsetOfNode::Kind::Field: {
+                    components.push_back(hl::OffsetOfNodeAttr::get(
+                        &mctx,
+                        mlir::StringAttr::get(
+                            &mctx, get_namespaced_decl_name(component.getField())
```

Comment:
Use symbol generator here. `get_namespaced_decl_name` is deprecated and will be removed.

Reply:
In fact you can remove it in this PR.